big_int.cpp

```diff
@@ -0,0 +1,519 @@
+
+#include <bits/stdc++.h>
+#define ll long long
+using namespace std;
+
+class BigInteger
+{
+public:
+string str;
+int sign;
+
+  BigInteger()
+  {
+    sign=0;
+    str="0";
```

Comment:
Codacy found an issue: [Variable 'str' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6290647)